### ISSUE TITLE
Different initial conditions to reduce shift in SWMT density?

### ISSUE THREAD

**Issue by schmidt-christina** [COSIMA/mom6-panan#43]
For whatever reason, the SWMT shifts to lighter densities when we start from ACCESS yr2 (closer to WOA), but the SWMT is quite stable when starting from  ACCESS yr200. 

Idea from last meeting: "Try some different initial conditions - We discussed [MIMOC](https://www.pmel.noaa.gov/mimoc/) yesterday, but looking at this now, I see it doesn't include seal data, which is the largest data type on the shelf."

I looked at what climatologies are available and agree that MIMOC is probably not the best as it doesn't include seal data and only includes data until 2011. I couldn't find any climatology that includes seal data, except for Pauthenet et al., 2021, but this covers only everything south of 60°S and the upper 400 m. Another global, full depth climatology is the "World ocean circulation experiment-argo global hydrographic climatology (WAGHC)" [(Gouretski, 2018, Ocean Science)](https://os.copernicus.org/articles/14/1127/2018/). It includes data until 2015 and an objective mapping method is used that preserves spatial gradients better compared to the method used for WOA.

I compared the salinity in January of all these climatologies at different depth levels: 
Overall, the climatologies seem to be mostly saltier in the 4 DSW regions compared WOA in the upper ~40m, but they are fresher than WOA below that. If we wanted to use another climatology WAGHC looks the best (saltier always in all 4 DSW regions in the upper 40 m, spans the whole water column down to 6650 m)

![image](https://user-images.githubusercontent.com/38216156/237032230-cab86793-2431-44dd-afc8-760fac9dceb5.png)

![image](https://user-images.githubusercontent.com/38216156/237032619-34ffc9bc-b9e7-4f67-a4e8-184d0ebbf1a5.png)

As no climatology looked saltier than WOA over the whole water column, it made me question whether we would see an effect at all. I then compared the different IC we are using at the moment (ACCESS yr 2 and yr 200) with WOA to get an idea about the magnitude of the difference in salinity and where they are occur in the water column. 

In the upper 100 m or even a bit further down, the salinity difference between yr2, yr200 and WOA is a lot higher than any difference between climatologies, so even if I use the WAGHC climatology as IC to increase the salinity in the upper ocean, the difference compared to WOA might not be big enough to see an effect. The difference between either yr2 or yr200 and WOA is also strong in the upper 200 m, only below 500 m yr2 and WOA are very similar and yr200 is fresher except for West Antarctica.

![image](https://github.com/COSIMA/mom6-panan/assets/38216156/03830f52-22b2-414b-80c7-fe3290a17e21)

In summary, the differences between different climatologies are not as big (especially other climatologies are not saltier than WOA below 40 m) as the differences between yr2, yr200 and WOA.

What to do next?

1.  Use WAGHC climatology as IC
or
2. Use modified ACCESS yr 2 IC where I increase the salinity everywhere (?) by 1 psu?
or
3. Something else?

**Comment by adele157** [comment]
Maybe give WAGHC a go? I agree they're not very different, but might be good to at least confirm it does nothing. We have hours available again on ol01 for testing.
